docker/docker-compose-embedded.yaml

```diff
@@ -1,5 +1,8 @@
 version: '3.7'
 
+volumes:
```

Comment:
Tricky part with elegant solution. 
In any real setup, the Lotus node and Powergate are in the same host.
Powergate saves the file locally and pass a file-path to Lotus to store data. So they have to share a docker-volume to make them appear running in the same host.